brave/src/test/java/com/linecorp/armeria/server/brave/BraveServiceIntegrationTest.java

```diff
@@ -65,34 +82,123 @@ protected void init() {
             return HttpResponse.of(HttpStatus.NOT_FOUND);
         });
         sb.service("/foo", (ctx, req) -> HttpResponse.of(OK, MediaType.PLAIN_TEXT_UTF_8, "bar"));
-        sb.service("/extra",
+        sb.service("/async", (ctx, req) -> asyncResponse(future ->
+            future.complete(HttpResponse.of(OK, MediaType.PLAIN_TEXT_UTF_8, "bar"))));
+
+        sb.service("/exception", (ctx, req) -> {
+            // TODO: HttpResponseException.of(status, "not ready")
```

Comment:
can we get a constructor like this please? it will help with testing to know difference between throwable message and http body.